pkg/ssh/manifest.json.in

```diff
@@ -0,0 +1,15 @@
+{
+    "version": "@VERSION@",
+    "requires": {
+	"cockpit": "134.x"
+    },
+    "priority": 1,
+    "bridges": [
+        {
+            "match": { "host": null },
+            "spawn": [ "@libexecdir@/cockpit-ssh" ],
+            "environ": [ "SSH_ASKPASS=@libexecdir@/cockpit-askpass" ],
```

Comment:
The environment doesn't get propagated across ssh by default ... that's both a blessing and a curse. Or have you seen this happen in real life? In which case it's not Cockpit specific: this would be a case with any ```SSH_ASKPASS``` usage anywhere (including stock ssh).